src/wallet/wallet.cpp

```diff
@@ -2353,7 +2352,8 @@ void CWallet::ResendWalletTransactions()
             // only rebroadcast unconfirmed txes older than 5 minutes before the
             // last block was found
             if (wtx.nTimeReceived > m_best_block_time - 5 * 60) continue;
-            if (wtx.RelayWalletTransaction(*locked_chain)) ++relayed_tx_count;
+            std::string unused_err_string;
+            if (wtx.SubmitMemoryPoolAndRelay(unused_err_string, true, *locked_chain)) ++relayed_tx_count;
```

Comment:
I guess we should recall it `submitted_count`  ? Wallet is just submitting to mempool and that's mempool job to pass it to P2P. We don't dissociate errors here anyways

Reply:
I don't care. It is only a log print, so can be done later. Though, I wanted to point out the change in behavior, since you tagged this with "refactor".

Reply:
Yes sorry for that, I modified the PR a bit since opening, so it's no more a refactor I guess..

Reply:
Lets do this in a follow-up, if important enough